locale/en.po

```diff
@@ -1595,48 +1607,52 @@ msgstr "Extras"
 msgid "CREDITS"
 msgstr "Credits"
 
-#: ../src/general/MainMenu.tscn:257
+#: ../src/general/MainMenu.tscn:254 ../src/general/PauseMenu.tscn:236
+msgid "QUIT_BUTTON_TOOLTIP"
+msgstr "Exit to desktop"
```

Comment:
We may have other platforms in the future, so the term "desktop" should be avoided here. Maybe "Exit the game" would be a good enough alternative?